app/common/commonMenu.js

```diff
@@ -28,6 +28,15 @@ const ensureAtLeastOneWindow = (frameOpts) => {
   }
 }
 
+const getCurrentWindowId = () => {
```

Comment:
maybe `currentWindow` should move to common and this code should go there? I'm going to add the active window to the appState and then everything can read it from the same place